Python/ceval.c

```diff
@@ -3351,10 +3350,13 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
                 block->b_level--;
             }
 
+            PyObject *_stack[4];
+            PyObject **stack = _stack + 1;  /* For PY_VECTORCALL_ARGUMENTS_OFFSET */
             stack[0] = exc;
             stack[1] = val;
             stack[2] = tb;
-            res = _PyObject_FastCall(exit_func, stack, 3);
+            res = _PyObject_Vectorcall(exit_func, stack,
+                    3 | PY_VECTORCALL_ARGUMENTS_OFFSET, NULL);
```

Comment:
How about this?

```c
PyObject *stack[4] = {NULL, exc, val, tb};
res = _PyObject_Vectorcall(exit_func, stack + 1, 3 | PY_VECTORCALL_ARGUMENTS_OFFSET, NULL);
```

Reply:
Fine for me. It's only slightly less efficient since you need to assign that `NULL`.

Reply:
> How about this?

OK, done.